android/cli/commands/_buildModule.js

```diff
@@ -1474,6 +1474,11 @@ AndroidModuleBuilder.prototype.packageZip = function (next) {
 					}.bind(this));
 				}
 
+				// respackageinfo file
+				if (this.manifest.respackage) {
```

Comment:
I'm not sure I understand. The `respackage` value is not a boolean key, it's an optional manifest value that typically is a java package name - and it's set manually by the user. We look to see if it's set to anything and if so, generate a `respackageinfo` file in the module holding the value.

So we either need to copy it from he manifest to the file expected, or change our app build process that handles modules to not look for this file but instead consult the manifest.